brave/browser-laptop
# Enabling one-click file open for a complete download (tray)

Fixes #13167

## Submitter Checklist:


## Test Plan:
1. Download a file
2. Move mouse to the generated download tray card
3. Click on the card, file should open
4. [Regression testing] Clicking on other download actions (such as delete, redownload, copy link) should continue to work as expected. 

## Reviewer Checklist:


Tests



Currently, when clicking on the downloaded file in the tray, a double click event was required for a download open to occur. This changes that event to a single click. A check is performed in the callback to ensure that the click target was not of type button, which would indicate another intended action (redownload, copy link, delete, etc)
